deploy/03-configure-farm/index.ts

```diff
@@ -0,0 +1,49 @@
+import { DeployerFn } from "@ubeswap/hardhat-celo"
+import { UbeswapFarmBot__factory, ERC20__factory } from "../../typechain"
+import { ContractTransaction } from "ethers"
+import ethers from "ethers"
+
+export const doTx = async (
+  action: string,
+  tx: Promise<ContractTransaction>
+): Promise<void> => {
+  console.log(`Performing ${action}...`);
+  const result = await (await tx).wait();
+  console.log(`${action} done at tx ${result.transactionHash}`);
+};
+
+const main: DeployerFn<{}> = async ({
+  deployer,
+}) => {
+
+  const FARM_ADDRESS = "0xc6686060A1BFa583566Ebca400A2C8771b20Cb8C"
+
+  const COMPOUNDER_ADDRESS = "0x642abB1237009956BB67d0B174337D76F0455EDd"
+  const COMPOUNDER_ROLE = await UbeswapFarmBot__factory.connect(FARM_ADDRESS, deployer).COMPOUNDER_ROLE()
+
+  await doTx(
+    "Setting compounder role",
+    UbeswapFarmBot__factory.connect(FARM_ADDRESS, deployer).grantRole(
```

Comment:
this file probly needs a similar warning about commenting out before compiling for the first time

Reply:
The only file you need to comment out is `/deploy/index.ts` I believe.

Reply:
oh that's good